administration/package.json

```diff
@@ -19,6 +20,8 @@
     "react": "^17.0.1",
     "react-apollo": "^3.1.5",
     "react-dom": "^17.0.1",
+    "react-router": "^5.2.0",
```

Comment:
I prefere the react-navigation router but probably it does not matter: https://www.npmjs.com/package/react-navigation